gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/GradleUtilTest.java

```diff
@@ -335,6 +359,31 @@ public void canBeResolved_DeprecatedAndNullResolutionAlternativesAndResolvable_s
     };
   }
 
+  private ScriptHandler createMockedBuildScript() {
+    final ModuleComponentIdentifier moduleComponentIdentifier = mock(ModuleComponentIdentifier.class);
```

Comment:
This part is hard to read, there is a lot nested mocks. Could this be simplified with `Mockito.RETURNS_DEEP_STUBS` ?

Reply:
Umm, do you mean using something like this?
```java
  private ScriptHandler mockedBuildScript() {
    return mock(ScriptHandler.class, RETURNS_DEEP_STUBS);
  }
```

I tried this but test is failing after this change. I'm getting this error:
```
java.lang.NullPointerException: plugins cannot be null
```